site/src/pages/release-notes/1.6.0.mdx

```diff
@@ -0,0 +1,315 @@
+             .build();
+  ```
+- You can now convert a stream of Protobuf Messages into
+  [JSON Text sequences](https://datatracker.ietf.org/doc/rfc7464/) using an annotated service. #3394
+  ```java
+  @Get("/items")
+  @ProducesJsonSequences
+  public Publisher<MyProtobufMessage> protobufJsonSeqPublisher() {
+      return StreamMessage.of(MyProtobufMessage.newBuilder()...build(),
+                              MyProtobufMessage.newBuilder()...build());
+  }
+
+  @Get("/items")
+  @ProducesJsonSequences
+  public Stream<MyProtobufMessage> protobufJsonSeqPublisher() {
+      return Stream.of(MyProtobufMessage.newBuilder()...build(),
+                       MyProtobufMessage.newBuilder()...build());
+  }
+  ```
+- You can now customize the default service name of a <type://RequestLog>. #3232 #3366
+  ```java
+  Server.builder()
+        .defaultServiceNaming(ctx -> {
+            final ServiceConfig config = ctx.config();
+            return config.route().patternString();
+       });
+  // For a specific service.
+  Server.builder()
+        .route().path("/")
+        .defaultServiceNaming(...)
+        ...
+  ```
+- You can now check if the current request is matched by any routes or not. #3365 #3378
+  ```java
+  ServerBuilder sb = ...
+  sb.decorator((delegate, ctx, req) -> {
+      if (ctx.config().route().isFallback()) {
+          // This request is not matched any routes.
+      }
+      return delegate.serve(ctx, req);
+  });
+  ```
+- The <type://BraveService> does not trace requests to the <type://TransientService> such as
+  <type://HealthCheckService> anymore. #3382
+  - You should specify <type://TransientServiceOption#WITH_TRACING> if you want to trace them.
+  ```java
+  HealthCheckService.builder()
+                    .transientServiceOptions(WITH_TRACING)
+                    ...
+                    .build();
+  ```
+- You can now clean up resources by overriding <type://FilteredStreamMessage#onCancellation(Subscriber)>
+  when `Subscription.cancel()` is called. #3375
+  ```java
+  new MyFilteredHttpResponse(res) {
+      ...
+      @Override
+      protected void onCancellation(Subscriber<? super U> subscriber) {
+          // Clean up resources.
+      }
+  }
+  ```
+- You can now easily set 'cookie' or 'set-cookie' headers. #3391
+  ```java
+  Cookie cookie = Cookie.of("cookie", "value");
+  RequestHeaders headers = RequestHeaders.builder(HttpMethod.GET, "/")
+                                         .cookie(cookie)
+                                         .build();
+
+  Cookie setCookie1 = ...
+  Cookie setCookie2 = ...
+  ResponseHeaders headers = ResponseHeaders.builder(HttpStatus.OK)
+                                           .cookies(setCookie1, setCookie2)
+                                           .build();
+  ```
+- You can now use `req.root_id` <type://BuiltInProperty> to log the id of <type://RequestContext#root()>.
+  #3429 #3433
+- You can now use Thrift 0.14.0. #3470 #3422
```

Comment:
?
```suggestion
- You can now use Thrift 0.14.1. #3470 #3422
```

Reply:
How about:

    You can now use Thrift 0.14.0 with new `armeria-thrift0.14` module.